src/views/home.html

```diff
@@ -48,8 +50,32 @@ <h2 class="govuk-heading-s">Repository requests</h2>
             <a class="govuk-link" href="/repo-request">Additional Repository Requests (e.g permission changes)</a>
           </li>
         </ul>
+
+        <form method="post">
+          {% if (add_team and add_team.length > 0)
+            or (add_repo and add_repo.length > 0)
+            or (add_member and add_member.length > 0) %}
+  
+            <br>
+            <br>
+            <h2 class="govuk-heading-l govuk-!-margin-0">What you have added so far:</h2>
+            <br>
+
+            {% include "include/list/added.html" %}
+
+            {{ govukButton({
+              text: "No more to add",
```

Comment:
Continue?

Reply:
I will update to `Continue` but has to be changed